website/src/components/content/CaseStudyPage.tsx

```diff
@@ -36,25 +36,25 @@ export const CaseStudyPage: React.FunctionComponent<Props> = ({
     <div className={`${slugify(customer).toLowerCase()}-${className}`}>
         <CaseStudyJumbotron className="mb-5" customer={customer} logo={logo}>
             {quote && <MediaQuote quote={quote.quote} author={quote.author} image={quote.image} />}
+            {pdf && (
+                <a
+                    href={pdf}
+                    className="btn btn-primary mt-4"
+                    rel="nofollow"
+                    onClick={() => {
+                        eventLogger.trackCaeStudyDownloadPDFClicked(customer)
```

Comment:
Just FYI this is misspelled `trackCaeStudyDownloadPDFClicked` -> `trackCaseStudyDownloadPDFClicked`. You can fix in another PR since it'll require renaming the variable everywhere.